### ISSUE TITLE
Anaconda python3.4 does not include proper library files

### ISSUE THREAD

**Issue by ProfPh** [ContinuumIO/anaconda-issues#559]
I created a new environment with version 3.4 in anaconda. Installed latest version of PyInstaller to that environment. Try to use PyInstaller to create an executable and it fails with:

        raise IOError(msg)
    OSError: Python library not found! 

It is failing to find the file: `libpython3.4.dylib` which can be included if `--enable-framework` is set (https://github.com/yyuu/pyenv/issues/443) but anaconda does not have this feature it seems.

**Comment by ccordoba12** [comment]
Python is not compiled as a framework to make conda environments structure similar to the Linux one. I think that also makes creating and deleting them easier.

Sorry, but I think that's not going to change in the near future.

**Comment by ProfPh** [comment]
Is there some way to manually grab those files and place them?

**Comment by ccordoba12** [comment]
Nop, you need to recompile Python as a framework yourself. It's not that hard (using conda-build), but I don't know if that could break other things in Anaconda (given the crucial importance of Python for conda).

I'd recommend you to not use Anaconda if you want to create self-contained apps in Mac. For example, I use Homebrew to create Spyder apps with py2app :-)

**Comment by ahmadia** [comment]
@ccordoba12 - I'm confused.  Don't certain things (native OS X matplotlib back end?) require a Framework build?  Do they just not work with Anaconda?

**Comment by ccordoba12** [comment]
Yes, they work just fine. I think for GUI stuff we have a little package called `python-app` that let's you run graphical apps without problems.

But I'm not sure how that works exactly. Pinging @ilanschnell for further insights.

----

By the way, my recommendation for not using Anaconda to create App bundles is because I tried to do that with py2app and found that it expects everything provided as a framework. I think Pyinstaller does the same.
